### PR TITLE
feat: adding TGeo digitization example

### PR DESCRIPTION
**Pull request by asalzburger**
This mini PR adds the `TGeoDigitizationExample` analog to all other examples in the `AcsExamples`.

### CODE REVIEW

**Comment by asalzburger** [comment]
Hey @niermann999 - this is in preparation for your QT, so I guess you could have a look. That's the smallest PR ever.

File: Examples/Run/Digitization/CMakeLists.txt [hunk 1]
```diff
@@ -14,5 +14,21 @@ target_link_libraries(
 
 install(
   TARGETS
-    ActsExampleDigitizationGeneric 
+    ActsExampleDigitizationGeneric
   RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
+
+# Digitization for Generic detector
```
Comment (niermann999) [inline Examples/Run/Digitization/CMakeLists.txt:20]: This is now TGeo, is it not?

**Review by niermann999 — APPROVED** [review]
Jep, that was a short read :)

**Comment by asalzburger** [comment]
Waiting for CI to run through.

**Pull request merged**